packages/OverflowMenu/test/overflowMenu.spec.js

```diff
@@ -101,5 +104,26 @@ describe("OverflowMenu", () => {
         expect(queryByText(/edit/i)).not.toBeInTheDocument();
       });
     });
+
+    it("should pass dynamic props to Confirmation on update", async () => {
+      expect(getByText(/confirm delete/i)).toHaveAttribute("aria-disabled", "false");
+
+      const newConfirmation = (
+        <Confirmation
+          isPending
```

Comment:
The exact case I've discovered. It's often needed to set confirm button state to `pending` (e.g. when making delete request to backend)